src/ScreenshotWindow.vala

```diff
@@ -117,7 +117,7 @@ public class Screenshot.ScreenshotWindow : Hdy.ApplicationWindow {
 
         var delay_spin = new Gtk.SpinButton.with_range (0, 15, 1);
 
-        var take_btn = new Gtk.Button.with_label (_("Take Screenshot"));
+        var take_btn = new Gtk.Button.with_label (_("Capture Screenshot"));
```

Comment:
I think I like "take" more here tbh. I'm not sure why but it feels more natural to me for some reason